package.json

```diff
@@ -33,26 +33,18 @@
     "yarn": ">= 1.3.0"
   },
   "scripts": {
+    "ci": "yarn run eslint && yarn run test:coverage",
     "clear": "rm -Rf tmp",
-    "test": "npm run mocha && npm run cucumber",
-    "test:coverage": "nyc --reporter=lcovonly --reporter=text-summary npm run test",
-    "test:mongo": "npm run eslint && cross-env NODE_ENV=test && nyc --reporter=lcovonly --reporter=text-summary npm run mocha:mongo",
+    "mongo": "mongod --dbpath data --quiet",
+    "mongo:win": "mongod --dbpath /data/db --quiet",
+    "test": "yarn run mocha && yarn run cucumber",
+    "test:coverage": "nyc --reporter=lcovonly --reporter=text-summary yarn run test",
     "eslint": "eslint server/. test/. features/. --config .eslintrc.json",
-    "start": "concurrently 'mongod' 'wait-on tcp:27017 && cross-env NODE_ENV=production node server/'",
-    "dev:debug": "npm run clear && concurrently '$npm_package_config_mongoDev &>/dev/null' 'wait-on tcp:27017 && cross-env DEBUG=feathers && cross-env NODE_ENV=development nodemon --inspect server/'",
-    "dev": "npm run clear && concurrently '$npm_package_config_mongoDev &>/dev/null' 'wait-on tcp:27017 && cross-env DEBUG=feathers && cross-env NODE_ENV=development nodemon server/'",
-    "dev:local": "sh scripts/run-local.sh",
```

Comment:
could you please re-add the dev:local command for mobile development?

Reply:
The script uses `ifconfig` to set some environment variables with the local IP address. E.g. I don't have `ifconfig` installed and the script fails silently. I find it much more convenient to set `host` in `config/local.json` to the local ip and just run `yarn run dev`. Can we keep `scripts/run-local.sh` deleted, please?

Reply:
I can add some instructions in `README.md` if you like.

Reply:
You missed the point of that script I think. It is getting your host ip to then start the app in your local network so you can access it from your mobile phone. It’s way more convenient then to write your ip into the local.config

But for me it’s fine to put it into the readme.